Gaijinizer patch/Map262.txt

```diff
@@ -5,16 +5,18 @@
 
 お前は先に進み続けた……
 > CONTEXT: Map262/events/2/pages/1/9/Dialogue/Gface
-Even though everyone stopped.........
-I knew it would be a bad result .........
-You kept moving forward ...#MTLed
+Even though everyone tried to stop you...
+And you knew it would lead to a bad end...
+
+You kept pushing on...#Ron
 > END STRING
```

Comment:
Tiny bit of rewording:
>_**Even though everyone tried to stop you...
Even though you knew it would lead to this bad
ending...
You kept pushing on.**_